documentation/de/getting_started_with_webdriver/third_party_drivers_and_plugins/index.html

```diff
@@ -11,6 +11,10 @@
     <title>Driverimplementierungen anderer Anbieter und Plugins :: Dokumentation für Selenium</title>
```

Comment:
Hi @boris779 ,
To be on the same page, this can be simple like 'Treiber und Plugins von Drittanbietern' to match other documents

Reply:
you are right